core/examples/luigi-sample-angular/e2e/tests/navigation.spec.js

```diff
@@ -88,6 +86,53 @@ describe('Navigation', () => {
     });
   });
 
+  describe('Node activation hook', () => {
```

Comment:
This is a cool scenario. But I think we should include clicking an actual node in the top or left navigation to activate the on-activation function as this is the main use case.

Reply:
Really good point, I have updated the example app and the e2e tests.